test/controllers/sign_offs_controller_test.rb

```diff
@@ -4,78 +4,90 @@ class SignOffsControllerTest < ActionController::TestCase
     assert sign_offs = assigns(:sign_offs)
-    assert_equal sign_offs(:sign_three), sign_offs.first
-    assert_equal sign_offs(:sign_one), sign_offs.last
+    sign_off_sort = @sign_offs.order('date_from' + ' ' + 'asc')
+    assert_equal 'approved', sign_offs.first.sign_off_status
+    assert_includes status, sign_offs.last.sign_off_status
   end
 
   test 'filter by sign_off_status column' do
-    xhr :get, :index, column_name: :sign_off_status, filter_by: :approved
+    prepare_sorting_data
+    user = User.first
+    xhr :get, :index, column_name: :sign_off_status, filter_by: :approved, access_token: user.access_token
     assert sign_offs = assigns(:sign_offs)
     sign_offs_expected = SignOff.where(sign_off_status: 'approved')
     assert_equal sign_offs_expected.sort, sign_offs.sort
-    xhr :get, :index, column_name: :sign_off_status, filter_by: :approved, sort: :date_from, direction: :desc
+    xhr :get, :index, column_name: :sign_off_status, filter_by: :approved, sort: :date_from, direction: :desc, access_token: @user.access_token
     assert sign_offs = assigns(:sign_offs)
-    assert_equal sign_offs(:sign_four), sign_offs.first
-    assert_equal sign_offs(:sign_five), sign_offs.second
+    assert_equal '2016-11-10', sign_offs.first.date_from.to_s
+    assert_equal '2016-11-10', sign_offs.last.date_from.to_s
   end
 
   test 'filter by date_from column' do
-    xhr :get, :index, column_name: :date_from, filter_by: '10/11/2016'
+    prepare_sorting_data
+    user = User.first
+    xhr :get, :index, column_name: :date_from, filter_by: '10/11/2016', access_token: user.access_token
     assert sign_offs = assigns(:sign_offs)
     sign_offs_expected = SignOff.where(date_from: '10/11/2016')
     assert_equal sign_offs_expected.sort, sign_offs.sort
-    xhr :get, :index, column_name: :date_from, filter_by: '10/11/2016', sort: :date_to, direction: :desc
+    xhr :get, :index, column_name: :date_from, filter_by: '10/11/2016', sort: :date_to, direction: :desc, access_token: @user.access_token
     assert sign_offs_sort = assigns(:sign_offs)
-    assert_equal sign_offs(:sign_two), sign_offs.first
-    assert_equal sign_offs(:sign_five), sign_offs.last
+    assert_equal 'approved', sign_offs.first.sign_off_status
+    assert_equal 'rejected', sign_offs.last.sign_off_status
   end
 
   test 'filter by date_to column' do
-    xhr :get, :index, column_name: :date_to, filter_by: '12/11/2016'
+    prepare_sorting_data
+    user = User.first
+    xhr :get, :index, column_name: :date_to, filter_by: '12/11/2016', access_token: user.access_token
     assert sign_offs = assigns(:sign_offs)
     sign_offs_expected = SignOff.where(date_to: '12/11/2016')
     assert_equal sign_offs_expected.sort, sign_offs.sort
-    xhr :get, :index, column_name: :date_to, filter_by: '12/11/2016', sort: :date_from, direction: :desc
+    xhr :get, :index, column_name: :date_to, filter_by: '12/11/2016', sort: :date_from, direction: :desc, access_token: @user.access_token
```

Comment:
Looks like this date is used multiple times. move it to variable and give a meaningful name.